plugins/statsd/statsd_input.go

```diff
@@ -152,15 +156,34 @@ func parseMessage(message []byte) ([]Stat, error) {
 		lines = [][]byte{message}
 	}
 
-	for _, line := range lines {
+	var num_lines int = len(lines)
+	var bad_lines_ct int = 0
```

Comment:
We don't really need to keep a separate count of the bad lines, do we? We can just check len(bad_lines) instead.